### ISSUE TITLE
Central Management - Save button does not work

### ISSUE THREAD

**Issue by krazykenny** [chocolatey/chocolatey-licensed-issues#99]
When attempting to either create a new computer or edit an existing computer listed in the management console, the "Save" button seems to be non functional. Clicking the Save button does nothing and only the Cancel button seems to work. This prevents any settings being saved - e.g. Setting a 'Friendly Name' for a computer. To do this at the moment I have to edit the SQL database directly to set it because the save button doesn't work.

The Save button doesn't seem to work for adding/editing Groups either.

**Comment by gep13** [comment]
@krazykenny thank you for taking the time to report this problem.  I have tested this, and I can confirm that this indeed isn't working as intended.  I have also tested this in the upcoming hotfix release of CCM, so it will be corrected in the next release.

Regarding the Groups section of CCM, this is actually not fully implemented in CCM 0.1.0, and showing the Group menu item in the CCM Dashboard is actually disabled by default.  If this has been changed to being shown, I would recommend that you turn this off again for the roles in question.
